src/components/TopBar.js

```diff
@@ -24,7 +26,7 @@ import { linkToJobManager } from 'src/pages/workspaces/workspace/JobHistory'
 const styles = {
   topBar: {
     flex: 'none', height: 66,
-    backgroundColor: colors.green[1], paddingLeft: '1rem',
+    background: `81px url(${headerLeftHexs}) no-repeat ${colors.green[1]}`, paddingLeft: '1rem',
```

Comment:
Since this is only used in two places, and it's overridden in one, why not just put it in the second?